### PR TITLE
Add information about pkgs_dir env variable

### PR DESCRIPTION
**Pull request by saraneem**
https://github.com/conda/conda-docs/issues/548

### CODE REVIEW

**Comment by saraneem** [comment]
cc @msarahan and @kalefranz

File: docs/source/user-guide/configuration/use-condarc.rst [hunk 1]
```diff
@@ -522,6 +522,28 @@ The CONDA_ENVS_PATH environment variable overwrites this setting:
...
+
+Specify directories in which packages are located. If this
+key is set, the root prefix ``pkgs_dir`` is not used unless
+explicitly included.
+
+EXAMPLE:
+
+.. code-block:: yaml
+
+  pkg_dirs:
```
Comment (msarahan) [inline docs/source/user-guide/configuration/use-condarc.rst]: please double check this.  Is it pkg_dirs, pkgs_dirs, pkgs_dir?  Extra s's matter a lot.  You have a lot of inconsistency here.

Comment (saraneem) [reply docs/source/user-guide/configuration/use-condarc.rst]: It is pkgs_dirs. Fixed those mistakes. 
Thank you!

**Comment by electronwill** [comment]
Good catch @msarahan ! These look great to me now. If no one else objects or merges it first, I'll merge it at the end of the day.

**Pull request merged**